<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>replenishments</class>
 <widget class="QWidget" name="replenishments">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Changes Plan</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="4">
    <widget class="QPushButton" name="r_add">
     <property name="text">
      <string>add</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QTableWidget" name="r_table"/>
   </item>
   <item row="2" column="2">
    <widget class="QDateEdit" name="r_date">
     <property name="displayFormat">
      <string>d/M/yy</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="r_sum"/>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QLabel" name="out_msg_3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="r_reset">
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
